apps/jetbrains/jetbrains.talon

```diff
@@ -14,11 +15,7 @@ smart: user.idea("action SmartTypeCompletion")
 (done | finish): user.idea("action EditorCompleteStatement")
 # Copying
 grab <number>: user.idea_grab(number)
-# Actions
-(action | please): user.idea("action GotoAction")
-(action | please) <user.text>:
```

Comment:
We probably want deprecation warning for the command `action [<user.text>]`

Reply:
I'm away this week but will make this change when I return.